lib/vanagon/platform/windows.rb

```diff
@@ -32,11 +32,14 @@ def package_name(project)
       end
 
       # Method to generate the files required to build a windows package for the project
+      # Both the MSI and nuget generic articfacts are created for simplicity.
       #
       # @param workdir [String] working directory to stage the evaluated templates in
       # @param name [String] name of the project
       # @param binding [Binding] binding to use in evaluating the packaging templates
       def generate_packaging_artifacts(workdir, name, binding)
+        # First the nuget artefacts (MAINT ticket coming up here to fix the spelling of the function name)
```

Comment:
the spelling of the function name? I think `generate_packaging_artifacts` is spelled correctly... it also need to stay as is because it's getting called at https://github.com/puppetlabs/vanagon/blob/master/lib/vanagon/driver.rb#L84